examples/tutorials/03_vision/04_exposure.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Demonstrate the manual and auto exposure settings of Cozmo's camera.
+
+This example demonstrates the use of auto-exposure and manual-exposure for
+Cozmo's camera. The current camera settings are overlayed onto the camera
+viewer window.
+'''
+
+
+import sys
+import time
+
+try:
+    from PIL import ImageDraw, ImageFont
+    import numpy as np
+except ImportError:
+    sys.exit('run `pip3 install --user Pillow numpy` to run this example')
+
+import cozmo
+
+
+# A global string value to display in the window to make it more obvious
```

Comment:
Might be nice to specify what window is implied (i.e., camera viewer window).

Also should specify that the program will update this global. Initially I was thinking that the user would add a setting on this line.